*** le601.py	2011-04-19 09:48:35.000000000 +0100
--- le601-exe.py	2011-04-19 09:48:35.000000000 +0100
***************
*** 157,165 ****
  WINDOWS_USAGE="""
  Windows specific commands:
    windowsevents 	follow the standard windows event logs 
-   installwinservice	installs the agent as a windows service 
- 	--no-defaults   do not follow standard windows logs automatically
-   monitorwinservice	starts monitoring as a windows service
  """
  
  
--- 157,162 ----
***************
*** 2565,2572 ****
  		'clean': cmd_clean,
  		# Windows
  		'monitorwinservice': cmd_win_service,
! 		'installwinservice': cmd_install_win_service,
! 		'windowsevents': cmd_follow_win_events,
  		# Clusters
  		'new-cluster': cmd_new_cluster,
  		'set-cluster': cmd_set_cluster,
--- 2562,2569 ----
  		'clean': cmd_clean,
  		# Windows
  		'monitorwinservice': cmd_win_service,
! 		#'installwinservice': cmd_install_win_service,
! 		#'windowsevents': cmd_follow_win_events,
  		# Clusters
  		'new-cluster': cmd_new_cluster,
  		'set-cluster': cmd_set_cluster,
