*** le	2012-06-28 11:46:58.000000000 +0100
--- le-exe2.py	2012-06-28 13:25:59.000000000 +0100
***************
*** 150,159 ****
  
  WINDOWS_USAGE="""
  Windows specific commands:
!   windowsevents 	follow the standard windows event logs 
!   installwinservice	installs the agent as a windows service 
! 	--no-defaults   do not follow standard windows logs automatically
!   monitorwinservice	starts monitoring as a windows service
  """
  
  
--- 150,157 ----
  
  WINDOWS_USAGE="""
  Windows specific commands:
!   windowsevents     follow the Application, System and Security win event logs 
!   windowsevents <eventlog>  follow the windows event log specified
  """
  
  
***************
*** 2771,2778 ****
  		'clean': cmd_clean,
  		'whoami': cmd_whoami,
  		# Windows
! 		'monitorwinservice': cmd_win_service,
! 		'installwinservice': cmd_install_win_service,
  		'windowsevents': cmd_follow_win_events,
  		# Clusters
  		#'new-cluster': cmd_new_cluster,
--- 2769,2776 ----
  		'clean': cmd_clean,
  		'whoami': cmd_whoami,
  		# Windows
! 		#'monitorwinservice': cmd_win_service,
! 		#'installwinservice': cmd_install_win_service,
  		'windowsevents': cmd_follow_win_events,
  		# Clusters
  		#'new-cluster': cmd_new_cluster,
